integration_tests/steps/two_sided_pay_as_clear.py

```diff
@@ -71,9 +66,24 @@ def test_cumulative_offer_bid_energy(context):
 @then('all traded energy have finite value')
 def test_finite_traded_energy(context):
     grid = context.simulation.area
-    for child in grid.children:
-        for a, b in child.dispatcher.interarea_agents.items():
-            for c in b:
-                for d in c.engines:
-                    for trade in d.markets.source.trades:
-                        assert isfinite(trade.offer.energy)
+    # Validate that all trades have less than 100 kWh of energy
+    assert all(trade.offer.energy < 100
+               for area in grid.children
+               for market in area.past_markets
+               for trade in market.trades)
+
+
+@then('there are files with offers, bids & market_clearing_rate for every area')
+def test_offer_bid_market_clearing_rate_files(context):
+    base_path = os.path.join(context.export_path, "*")
+    file_list = [os.path.join(base_path, 'grid-offers.csv'),
+                 os.path.join(base_path, 'grid-bids.csv'),
+                 os.path.join(base_path, 'grid-market-clearing-rate.csv'),
+                 os.path.join(base_path, 'grid', 'house-1-offers.csv'),
+                 os.path.join(base_path, 'grid', 'house-1-bids.csv'),
+                 os.path.join(base_path, 'grid', 'house-1-market-clearing-rate.csv'),
+                 os.path.join(base_path, 'grid', 'house-2-offers.csv'),
+                 os.path.join(base_path, 'grid', 'house-2-bids.csv'),
+                 os.path.join(base_path, 'grid', 'house-2-market-clearing-rate.csv')]
+
+    assert all(len(glob.glob(f)) == 1 for f in file_list)
```

Comment:
How about adding a check if the file is not empty, like @spyrostz did in his last PR:
`assert all(len(open(glob.glob(f)[0]).readlines()) > 0 for f in file_list)` ?

Reply:
I deliberately didn't added this check as it is not necessary to have market_clearing_rate

Reply:
Hmm, ok. So there the simulation returns an empty file in this case?

Reply:
Yes

Reply:
Ok, Thanks!

Reply:
The file should not be exported at all in case of pay-as-bid and one sided market. The reason for that is that even now there are quite a lot of files that are exported, and it would help reduce the clutter if the files were not exported at all in that case.

Reply:
What is the status on this issue? I dont see a check for empty file. So the market_clearing_rate is exported every time, yes?

Reply:
For now yes, it would be imported everytime.